src/pages/ChampionDetails.jsx

```diff
@@ -21,23 +23,110 @@ function ChampionDetails() {
     };
 
     fetchChampion();
-
     return () => {
       abortController.abort();
     };
   }, [name]);
 
+  const handleSpellClick = spell => {
+    setSelectedSpell(spell);
+    setSelectedPassive(false);
+  };
+
+  const handlePassiveClick = () => {
+    setSelectedSpell(false);
```

Comment:
```suggestion
    setSelectedSpell(null);
```
Pas génial de changer de type de données (passer de `string` à `boolean`)